lib/suspenders/app_builder.rb

```diff
@@ -1,6 +1,11 @@
+require 'forwardable'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.